.github/workflows/search-workflow.yml

```diff
@@ -36,17 +36,13 @@ jobs:
       run: bash -e examples/search/bank.sh
     - name: generating statistic files
       run: bash -e examples/search/stats.sh
-    - name: generating workflow
```

Comment:
At least until we remove submit_dax can we keep one test workflow using it? The template bank workflow is already testing "submit_now"

Reply:
Ok, I've reverted this